client/lib/sites-list/test/test-log-store.js

```diff
@@ -1,26 +1,35 @@
-var assert = require( 'assert' ),
-	actions = require( 'lib/mock-actions' );
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
 
-describe( 'Sites Log Store', function() {
-	var Dispatcher, LogStore;
+/**
+ * Internal dependencies
+ */
+import useMockery from 'test/helpers/use-mockery';
 
-	beforeEach( function() {
+describe( 'Sites Log Store', () => {
+	let Dispatcher, LogStore, actions;
+
+	useMockery();
```

Comment:
Hopefully we migrate to Redux someday and solve that. Better don't waste time and keep it as it is :) I trust @blowery on this one. I I remember properly there is some magic that `useMockery` uses to make sure we always have clean state of Dispatcher.